CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make by opening an issue. Discussing the design before implementing a big
+feature can result in a much better and cleaner implementation. 
+
+## Pull Request Checklist
+
+Please ensure your PR meets the following requirements:
+- Commits MUST be signed-off by the committer. For example:
```

Comment:
I think that the capital "MUST" looks a bit stern, especially in this context, which is not an RFC. I'd suggest we switch it to lowercase. If you prefer the capital one or if you've seen other projects using that, then ok, let's go with that.

Reply:
You are correct that I capitalized it because I was following the RFC style.
I like the RFC style because it makes requirements more clear, but I get why it might put some people off, especially if they are not familiar with this style.
I will change it to lowercase, for the sake of better communication.